tap-setup-scripts/src/inputs/tap-values-build.yaml

```diff
@@ -0,0 +1,38 @@
+#@ load("@ytt:data", "data")
+---
+profile: build
+ceip_policy_disclosed: true # Installation fails if this is set to 'false'
```

Comment:
a question came up about ceip. do we create the configmap to enable it?

Reply:
I am not sure about this.